dirsrvtests/tests/suites/vlv/regression_test.py

```diff
@@ -170,6 +179,120 @@ def cleanup(inst):
     remove_entries(inst, DEFAULT_SUFFIX, "(cn=testuser*)")
 
 
+@pytest.fixture
+def vlv_setup_with_two_backend(topology_st, request):
+    inst = topology_st.standalone
+    belist = []
+
+    def fin():
+        # Cleanup function
+        if not DEBUGGING and inst.exists() and inst.status():
+            for be in Backends(inst).list():
+                if be.get_attr_val_utf8_l('cn') in [ 'be1', 'be2' ]:
+                    be.delete()
+
+    request.addfinalizer(fin)
+
+    def setup_vlv_and_backend(inst, bename):
+        # Create a backend, add vlv index and populate the backend.
+        bes = Backends(inst)
+        suffix = f'o={bename}'
+        be = bes.create(properties={
+            'cn': bename,
+            'nsslapd-suffix': suffix,
+        })
+        # Add suffix entry
+        Organization(inst, dn=suffix).create(properties={ 'o': bename, })
+        # Configure vlv
+        vlv_search, vlv_index = create_vlv_search_and_index(
+            inst, basedn=suffix,
+            bename=bename, scope=1,
```

Comment:
Same here regarding the scope.

Reply:
OK, will use ldap.SCOPE_ONELEVEL instead of 1